### PR TITLE
Move some categories classified as trades to ledger actions at crypto.com

### LINKED ISSUE #4039: Crypto.com cashback and earn rewards appear as "buy" after file import
As described in the subject, after importing transactions from the mobile app csv, all CRO cashback appear as "buy". Likewise, for the weekly rewards from Crypto Earn (when you lock some crypto for specific periods and get in-kind rewards).

Rotki version: 1.23.2

### PR DESCRIPTION
**Pull request by yabirgb**
Affected events:

- referral_card_cashback
- referral_gift
- referral_bonus
- crypto_earn_interest_paid

Closes #4039

### CODE REVIEW

File: rotkehlchen/tests/utils/dataimport.py [hunk 1]
```diff
@@ -434,6 +430,8 @@ def assert_cryptocom_import_results(rotki: Rotkehlchen):
         link=None,
         notes=get_cryptocom_note('Refund from Merchant XXX'),
     )]
+    print(expected_ledger_actions)
```
Comment (LefterisJP) [inline rotkehlchen/tests/utils/dataimport.py:433]: Ehm ...

**Review by LefterisJP — APPROVED** [review]
Not really sure if I can test this in any way. If the user is satisfied with the results then I guess it works.

**Comment by yabirgb** [comment]
@LefterisJP I didn't send it to the user. Just we had the data and adapted them to be income actions instead of trades as they should be

**Comment by LefterisJP** [comment]
> @LefterisJP I didn't send it to the user. Just we had the data and adapted them to be income actions instead of trades as they should be

Well sounds good then? I can't tell you since I don't have any data to compare. But yes from the names they do not sound liek trades, but as ledger actions.

**Comment by cattaneod** [comment]
Now cashback appear as "income", which is not ideal: income is taxable while cashback is non-taxable in many countries.

**Comment by LefterisJP** [comment]
> Now cashback appear as "income", which is not ideal: income is taxable while cashback is non-taxable in many countries.

Good point @cattaneod. All these need to be customizable, otherwise what one user wants will not work for another and vice versa.

**Pull request merged**